### DISCUSSION TITLE
how to manage local data and online database together in electron app?

### DISCUSSION THREAD

**Discussion by luxuereal** [luxuereal/budget_nextron_sqlite#17] [Q&A] [answered]
Hello, everyone.
Let us brainstorm.
Let us discuss how to manage local data and online database in electron app?

**Comment by pearlights** [comment]
> Hello, everyone. Let us brainstorm. Let us discuss how to manage local data and online database in electron app?

Yeah, I have ever used SQLite in electron app.

**Comment by vik-splav** [comment]
Well, it is good, But SQLite is only for offline.
In case of offline, I prefer NxDB.

**Reply by luxuereal** [reply]
Oh, It is great. Do you know how to use that?

**Reply by pearlights** [reply]
Yeah, a little.

**Reply by vik-splav** [reply]
good, please show us.

**Comment by pearlights** [comment, marked as answer]
Okay. I share the docs.
https://rxdb.info/quickstart.html

**Reply by luxuereal** [reply]
good.

**Reply by vik-splav** [reply]
cool
